### ISSUE TITLE
How do I select another theme css

### ISSUE THREAD

**Issue by appernetic** [mmrath/hugo-bootstrap#3]
I tried to change the theme from default to paper. I saw in the partials that the variable is Site.Params.theme.name.

In settings I tried with
[params]
theme = "paper"

and also

[params]

[theme]
name = "paper"

and my last try:

under [params]
theme.name = "paper"

What is the right combination?

Last test with: theme = "paper.css" under [params]
Error:

``` html
ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in theme/partials/base/imports.html ERROR: 2016/02/13 template: theme/partials/base/imports.html:8:69: executing "theme/partials/base/imports.html" at <.Site.Params.theme.n...>: can't evaluate field name in type interface {} in ```

**Comment by appernetic** [comment]
I think you have a bug in the layouts/partial/base/imports.html file:

    <link rel="stylesheet" href="{{ "/css/theme/" | absURL }}{{ .Site.Params.theme.name }}.css">

It should be:

<link rel="stylesheet" href="{{ "/css/theme/" | absURL }}{{ .Site.Params.theme}}.css">

Then it is possible to add under:
[partials]
theme = "paper"

**Comment by mmrath** [comment]
@appernetic Did you try? If not, could you try and let me know if it works?
```
[params.theme]
  name = "cosmo"
```

**Comment by mmrath** [comment]
I am closing this, feel free to reopen if the issue is not yet resolved

**Comment by appernetic** [comment]
Hi, thanks,
It works! I hadn't got time to test your last suggestion before now.
I have now forked your theme and did small changes in the styles.css so that the right sidebar is not to close to the top menu when using the Paper theme. I also added a lite more space between the bottom and the copyright text.

To have my users understand how it works I also added instructions to the READMe.md file and added a new screenshot.

Other than that my [fork](https://github.com/appernetic/hugo-bootstrap-mod) will work both in Hugo and Appernetic.io
